### PR TITLE
Create pokescape-pvm

### PR DESCRIPTION
**Pull request by Quoded**
The companion plugin for PokeScape PvM. This plugin automatically captures and submits screenshots for leveling Runemon, battling Gyms and other activities while playing PokeScape.

### CODE REVIEW

**Comment by Nightfirecat** [comment]
Since this plugin always submits requests to api.pokescape.com, please add a `warning` to the plugin indicating that it will make outbound requests. ([see example](https://github.com/runelite/plugin-hub/blob/0cfd0e2c09b4bd88cc0f58cb9677c0a4f123b872/plugins/neverscapealone#L3)) This warning should ensure users are warned that their IP addresses, RSNs, world, and loot will be sent.
LGTM otherwise

**Comment by Quoded** [comment]
> Since this plugin always submits requests to api.pokescape.com, please add a `warning` to the plugin indicating that it will make outbound requests. ([see example](https://github.com/runelite/plugin-hub/blob/0cfd0e2c09b4bd88cc0f58cb9677c0a4f123b872/plugins/neverscapealone#L3)) This warning should ensure users are warned that their IP addresses, RSNs, world, and loot will be sent. LGTM otherwise

I configured the installation warning [here](https://github.com/Quoded/PokeScape-PvM/blob/baeb6822e5aa73b8adb6d7c55607d88188c955df/runelite-plugin.properties#L6) inside runelite-plugin.properties. Does that work or does it need to be configured elsewhere?

**Comment by raiyni** [comment]
The warning goes in the plugin file you are submitting here.

**Comment by Quoded** [comment]
Updated plugin file here with the warning. Lmk if that looks ok

File: plugins/pokescape-pvm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+repository=https://github.com/Quoded/PokeScape-PvM.git
+commit=baeb6822e5aa73b8adb6d7c55607d88188c955df
+warning=This plugin submits the names of you, loot, screenshots and other character data to api.pokescape.com
+authors=Quo
```
Comment (Nightfirecat) [inline plugins/pokescape-pvm:4]:
```suggestion
warning=This plugin submits your IP address, RSN, world, loot, and screenshots to a 3rd party website not controlled or verified by the RuneLite Developers.
```

The `authors` line is not needed in this file, as it is only used here to help validate whether other users are authorized to submit plugin updates on your behalf.

**Review by Nightfirecat — CHANGES_REQUESTED** [review]

File: plugins/pokescape-pvm (hunk 1 above)
Comment (Quoded) [reply plugins/pokescape-pvm]: Done

**Pull request merged**